src/reader/redis.clj

```diff
@@ -0,0 +1,27 @@
+(ns reader.redis
+  (:require [taoensso.carmine :as car]
+            [reader.config :as c]))
+
+(def server-conn {:pool {} :spec {:host (c/config :redis-host)}})
+
+(defmacro wcar*
+  [& body]
+  `(car/wcar server-conn ~@body))
+
+(defn set-html
+  [url html]
+  (wcar* (car/set url html)
+         (car/expire url 300)))
+
+(defn get-html
```

Comment:
Might be better to separate the `redis` namespace with the domain for which you're applying the cache.

One potential design could could be holding the `server-conn` in an atom or a connection pool, and using that to do what appears to be effectively page caching and all page-caching related fns can sit in a single `page-cache` ns. 

This will help later if you need to put UUIDs in the cache key (or mimic [russian-doll caching](https://guides.rubyonrails.org/caching_with_rails.html#russian-doll-caching)), or set other page-caching related domain logic.

You can make a judgment call based on how much work that is, since it's still early days of the project.

Reply:
+1 to this.